performance/performance_approach.rst

```diff
@@ -1,7 +1,5 @@
-.. _vonk_performance:
-
-Performance of Firely Server
-========================================
+Performance of Firely Server Approach
```

Comment:
Let's go with 'Performance of Firely Server' or 'Performance Approach of Firely Server'. 'Firely Server Approach' grammatically reads like Firely Server is a plane on its approach to a runway hehe

Reply:
Agree! :-)